setup.py

```diff
@@ -37,6 +37,12 @@
             "Werkzeug",
             "zope.interface",
         ],
+        extra_requires={
+            "docs": [
+                "Sphinx==3.5.1",
+                "sphinx-rtd-theme==0.5.1",
+            ]
+        },
```

Comment:
Is this a thing that RTD uses?
I'm having a hard time understanding why a user of the Klein package would want an extra that installs Sphinx.

Reply:
Ah I see it above… ok, I guess.

Reply:
this change is to have all dependencies in a single place ... rather than split between setup.py and tox

But this can be reverted as it's not a blocker for read the docs.

Reply:
I reverted for now. Maybe a `requirements-docs.txt` file?